django_rq/utils.py

```diff
@@ -10,9 +10,42 @@
 from rq.worker import Worker
 from rq.worker_registration import clean_worker_registry
 
-from .queues import get_connection, get_queue_by_index
+
+from .queues import get_connection, get_queue_by_index, get_scheduler
 from .settings import QUEUES_LIST
 from .templatetags.django_rq import to_localtime
+from django.core.exceptions import ImproperlyConfigured
+
+def get_scheduler_pid(queue):
+    '''Checks whether there's a scheduler-lock on a particular queue, and returns the PID.
+        It works by first checking if there's an RQ-Scheduler active.
+        If not, it checks the RQ's RQScheduler for a scheduler lock in the desired queue
+        Note: result might have some delay (1-15 minutes) but it helps visualizing whether the setup is working correcly
+    '''
+    try:
+        if not queue:
+            raise ValueError("queue argument not defined")
```

Comment:
We can remove this check since `queue` is already a required argument.

Reply:
Great! Done.